src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Color[] colors = Color.values();
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
+        return colors[random.nextInt(colors.length)];
+    }
```

Comment:
Return color in String type

Reply:
why?

Reply:
Returning Color leaves more options to users of this class, if a consumer wants string representation, he can just call `.name()`, but we should force someone to parse String into a constant if there ever will be a need for it. Flexibility and 'future-proofness' are important considerations, so better leave as is

Reply:
thank you very much for the explanation!